src/app/action/login.ts

```
@@ -0,0 +1,145 @@
+'use server';
+
+import { Import } from "lucide-react";
```

ESLint

[error] 'Import' is defined but never used.

(@typescript-eslint/no-unused-vars)